tests/cpp_unit.cpp

```diff
@@ -2146,6 +2146,18 @@ DEFINE_TEST(test_cpp_contains_range_interleaved_containers) {
     roaring.containsRange(0x1FFFF, 0x2FFFF + 2);
 }
 
+DEFINE_TEST(test_cpp_copy_map_iterator_to_different_map) {
```

Comment:
Could you add a brief comment about the purpose of the test?

Reply:
Done